src/Invoice_GenUI/Invoice_GenUI/Models/Services/InvoiceListService.cs

```diff
@@ -1,25 +1,27 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Net.Http;
-using System.Net.Http.Json;
+﻿using System.Collections.Generic;
 using System.Threading.Tasks;
 
 namespace Invoice_GenUI.Models.Services
 {
-    public class InvoiceListService : IInvoiceListService
+    public class InvoiceListService : BaseService, IInvoiceListService
     {
         public async Task<List<InvoiceModel>> GetInvoices()
         {
-            using (var client = new HttpClient())
+            return await SendHttpRequest<List<InvoiceModel>>("Invoice") ?? new();
+        }
+        public async Task<bool> DeleteInvoice(int id)
```

Comment:
:question: if there is a base service which wraps the HttpClient, is it worth creating a generic version of this method too?